cmd/device-worker/main.go

```diff
@@ -23,6 +23,10 @@ import (
 
 var yggdDispatchSocketAddr string
 
+const (
+	defaultDataDir = "/var/local/yggdrasil"
```

Comment:
Is there a need to update PoC docs about this ?

Reply:
Good point. Yes.